docs/content/getting-started/hello-dagster.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Hello Dagster | Dagster
+description: Run dagster for the first time
+---
+
+# Hello Dagster
+
+This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories.
```

Comment:
I like that the intro tells me what I can expect to have by the end, but this doesn't draw me in. How long will this take me? Why should I do it?

Reply:
```suggestion
This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories. In less than 5 minutes you will be able to develop functional code and explore dagster's user interface.
```